stm32_base/hal/PWM_stm32/PWM_device_stm32.c

```diff
@@ -37,6 +41,8 @@ static gpio analogCtrlGpios[] = {
 	{ RCC_AHB1Periph_GPIOE, GPIOE, GPIO_Pin_1}
 };
 
+static uint16_t g_currentClockPeriod = 0;
```

Comment:
the `= 0` is not needed here as that is the default value of static variables as defined per BSS rules on ELF files.  The = 0 makes the firmware larger for no good reason (albeit a very small size increase).